linera-sdk/src/contract/conversions_to_wit.rs

```diff
@@ -25,6 +25,28 @@ impl From<CryptoHash> for wit_system_api::CryptoHash {
     }
 }
 
+impl From<Owner> for wit_system_api::CryptoHash {
+    fn from(owner: Owner) -> Self {
+        let parts = <[u64; 4]>::from(owner.0);
+
+        wit_system_api::CryptoHash {
+            part1: parts[0],
+            part2: parts[1],
+            part3: parts[2],
+            part4: parts[3],
+        }
```

Comment:
Isn't that already implemented?

```suggestion
        wit_system_api::CryptoHash::from(owner.0)
```